Description: Update .desktop file with correct paths, category and line endings
Author: Iain R. Learmonth <irl@fsfe.org>
Last-Update: 2014-09-08

--- direwolf.orig/direwolf.desktop
+++ direwolf/direwolf.desktop
@@ -1,10 +1,9 @@
-[Desktop Entry]
-Type=Application
-Exec=lxterminal -t "Dire Wolf" -e "/usr/local/bin/direwolf"
-Name=Dire Wolf
-Comment=APRS Soundcard TNC
-Icon=/usr/share/direwolf/dw-icon.png
-Path=/home/pi
-#Terminal=true
-Categories=HamRadio
-Keywords=Ham Radio;APRS;Soundcard TNC;KISS;AGWPE;AX.25
\ No newline at end of file
+[Desktop Entry]
+Type=Application
+Exec=/usr/bin/direwolf
+Name=Dire Wolf
+Comment=APRS Soundcard TNC
+Icon=/usr/share/direwolf/dw-icon.png
+Terminal=true
+Categories=Network;HamRadio
+Keywords=Ham Radio;APRS;Soundcard TNC;KISS;AGWPE;AX.25
